module/src/components/singleSelect/singleSelect.basic.scss

```diff
@@ -0,0 +1,5 @@
+@use "../../theme/variables.scss";
+
+.arm-single-select-input {
+  margin: variables.$spacing-xsmall 0;
+}
```

Comment:
is adding margin too opinionated for the basic stylesheet? I would argue yes but might be one for discussion.